ui/src/components/DashboardMetrics.js

```diff
@@ -0,0 +1,234 @@
+import React, { useCallback } from 'react';
+import { useQuery } from 'react-query';
+import { LineChart, Loader } from '@scality/core-ui';
+import { lighten, darken } from 'polished';
+
+import { refreshNodesAction, stopRefreshNodesAction } from '../ducks/app/nodes';
+import { useRefreshEffect, fromUnixTimestampToDate } from '../services/utils';
+import { useTypedSelector } from '../hooks';
+import {
+  queryNodeCPUMetrics,
+  queryNodeMemoryMetrics,
+  queryNodeLoadMetrics,
+} from '../services/prometheus/fetchMetrics';
+import { LAST_TWENTY_FOUR_HOURS, LAST_ONE_HOUR } from '../constants';
+import {
+  yAxisUsage,
+  yAxis,
+  getTooltipConfig,
+} from '../components/LinechartSpec';
+import {
+  GraphTitle,
+  GraphWrapper,
+} from '../components/style/CommonLayoutStyle';
+import { useDynamicChartSize } from '../services/utils';
+
+// import styled from 'styled-components';
+
+const DashboardMetrics = () => {
+  // Making sure nodes list is loaded
+  useRefreshEffect(refreshNodesAction, stopRefreshNodesAction);
+  const nodes = useTypedSelector((state) => state.app.nodes.list);
+  const theme = useTypedSelector((state) => state.config.theme);
+  // Get dynamic chart size for 1 column, 4 rows
+  const [graphWidth, graphHeight] = useDynamicChartSize(
+    'dashboard-metrics-container',
+    1,
+    4,
+  );
+
+  const metricsTimeSpan = LAST_TWENTY_FOUR_HOURS;
+
+  // We use 4 main color from the palette and decline them (lighter/ darker) when we have more than 4 datasets
+  const colorRange = [
+    '#245A83',
+    '#808080',
+    '#A04EC9',
+    '#C6B38A',
+    lighten(0.3, '#245A83'),
+    lighten(0.3, '#808080'),
+    lighten(0.3, '#A04EC9'),
+    lighten(0.3, '#C6B38A'),
+    darken(0.2, '#245A83'),
+    darken(0.2, '#808080'),
+    darken(0.2, '#A04EC9'),
+    darken(0.2, '#C6B38A'),
+  ];
+
+  const xAxis = {
+    field: 'date',
+    type: 'temporal',
+    axis: {
+      // Refer to all the available time format: https://github.com/d3/d3-time-format#locale_format
+      format:
+        metricsTimeSpan === (LAST_ONE_HOUR || LAST_TWENTY_FOUR_HOURS)
+          ? '%H:%M'
+          : '%m/%d',
```

Comment:
From the screenshot, I see the x-axis is always the same date, not sure what's the timespan though. Please add the timespan selector with three choices.

Reply:
For now it's static on LAST_TWENTY_FOUR_HOURS. The design does not show a timespan selector so I figured it would be static.

@ChengYanJin @Cuervino there is not much space between the panel title and the Advanced metrics button on the design, where should I put this ?